samples/Nancy.Demo.Hosting.Kestrel/HomeModule.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Nancy.Demo.Hosting.Kestrel
+{
+    using System.Threading.Tasks;
+
+    public class HomeModule : NancyModule
+    {
+        public HomeModule()
+        {
+            Get["/"] = (o, token) => Task.FromResult<dynamic>("Hello from Nancy running on CoreCLR");
+
+            Get["/conneg/{name}"] = (parameters, token) => Task.FromResult<dynamic>(new Person() { Name = parameters.name });
+        }
+    }
+
+    public class Person

```

Comment:
If the Person is a response object then it should be nested in the HomeModule, for name spacing, in a real world app, because that response object shouldn't be shared among other responses.